### PR TITLE
Add an option to lock computer

### PR DESCRIPTION
**Pull request by pierre42100**
It would be awesome if this cmatrix could be used as a lock screen...
Ways to lock screen:
* Use the -L option
* Press the L key while the application is running...

It also shows a message while locked and it catch Ctrl+C and Ctrl+Z...

### CODE REVIEW

File: cmatrix.c [hunk 1]
```diff
@@ -708,6 +724,34 @@ if (console) {
...
+            move(msg_x, msg_y-2);
+            addch(' ');
+            addch(' ');
+            addstr(msg);
+            addch(' ');
+            addch(' ');
+
+            //Add space after message
+            move(msg_x+1, msg_y-2);
+            for(int i = 0; i < strlen(msg)+4; i++)
```
Comment (abishekvashok) [inline cmatrix.c]: Use c99 style for declaration here pls.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by pierre42100** [comment]
I think I solved the issue. If not, do not hesitate to tell me...

File: cmatrix.c (hunk 1 above)
Comment (pierre42100) [reply cmatrix.c]: I think I solved the issue. If not, do not hesitate to tell me...

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
Thanks!

**Pull request merged**